lib/hocon/config_util.rb

```diff
@@ -0,0 +1,17 @@
+require 'hocon/impl/config_impl_util'
+
+
+# Contains static utility methods
+class Hocon::ConfigUtil
+  def self.quote_string(string)
```

Comment:
This is a top-level, public-facing class, so I think we should add the comments from the upstream library to each of these methods.